File: athrill-account-client/projects/at-models/src/lib/at-logs/record-log.model.ts
import { BaseModel } from '../at-base.model';
import { ATEntityModel } from '../system-values/at-entity.model';

export class RecordLogModel extends BaseModel {
    public RecordLogId: number;
    public RecordType: number;
    public EntityId: string;
    public Record: string;
    public CreatedBy: number;
    public IPAddress: string;
    public ClientName: string;
    public Id: number;

    public Entity : ATEntityModel;
}

File: athrill-account-server/Libraries/Model/ATLogs/RecordLogModel.cs
using AT.Model.SystemValues.ATEntities;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.ATLogs
{
    public class RecordLogModel
    {
        public long RecordLogId { get; set; }
        public int RecordType { get; set; }
        public int EntityId { get; set; }
        public string Record { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }
        public string IPAddress { get; set; }
        public string ClientName { get; set; }
        public long Id => RecordLogId;
        public ATEntityModel Entity { get; set; }

        [JsonIgnore]
        public Guid InsertId { get; set; }
        [JsonIgnore]
        public Guid BatchId { get; set; }
    }
}
